src/renderer/src/views/components/AiTab/index.less

```
@@ -761,6 +764,23 @@
   }
 }
 
+:global(.history-dropdown-motion-enter),
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)